browser/net/url_context.cc

```diff
@@ -98,7 +98,7 @@ std::shared_ptr<brave::BraveRequestInfo> BraveRequestInfo::MakeCTX(
     // given that this is always empty for subresources.
     ctx->network_isolation_key =
         request.trusted_params->isolation_info.network_isolation_key();
-    ctx->tab_origin = ctx->network_isolation_key.GetTopFrameSite()
+    ctx->tab_origin = request.trusted_params->isolation_info.top_frame_origin()
```

Comment:
`trusted_params` is `base::Optional` - should maybe check if it's set?

Reply:
we are already inside this check :)

Reply:
Ah, and so we are 👍